### ISSUE TITLE
[BUG] Desynchro between the state of crc and podman desktop reported status

### ISSUE THREAD

**Issue by slemeur** [crc-org/crc-extension#160]
I've started with `crc start` from the cli and the environment is not visible in Podman Desktop.

<img width="1589" alt="Capture d’écran 2023-06-22 à 10 49 30" src="https://github.com/crc-org/crc-extension/assets/1636769/3770abb0-be77-4ade-969d-1fccae6d810b">

### Logs

Before gather the logs try following if that fix your issue
```bash
$ crc delete -f
$ crc cleanup
$ crc setup
$ crc start --log-level debug
```

See the logs:
<img width="1598" alt="Capture d’écran 2023-06-22 à 10 50 53" src="https://github.com/crc-org/crc-extension/assets/1636769/f4fa7928-9273-4a93-9214-ae26b9e68879">

**Comment by gbraad** [comment]
the init of the extension only happens on start of the extension. this determines the existence. to solve this would mean we regularly have to test for this on interval.

does this happen also for any of the other extensions?

**Comment by deboer-tim** [comment]
Yes, other extensions either have a listener or periodically poll for state.

**Comment by gbraad** [comment]
We are in the process of adding SSE to the backend. This will help to resolve this; and it will prevent unnecessary polling (uses a listener).

**Comment by danivilla9** [comment]
The issue definitely persists. Pictures from extension on main branch:

<img width="1272" height="830" alt="Image" src="https://github.com/user-attachments/assets/4d17d5e2-a9d1-40c5-aff5-945a62f7b138" />

<img width="1272" height="830" alt="Image" src="https://github.com/user-attachments/assets/f31a6f2b-874d-44c1-9383-754c59827973" />

Re-starting PD prompts the correct status.
